web/src/modules/topic/utils/layout.ts

```diff
@@ -20,12 +20,10 @@ export const layout = async (nodes: Node[], edges: Edge[], orientation: Orientat
     "elk.layered.spacing.nodeNodeBetweenLayers": orientation === "DOWN" ? "90" : "135",
```

Comment:
this also needs to be updated. i played with some numbers and this is what I think looks decent at default font size 16. pls replace the existing `// not sure why larger...` comment on the line before with the comment I added here, so that it's clear that the 90 isn't only correlated to the node height (I thought it was at first)
```suggestion
    // these spacings are just what roughly seem to look good
    "elk.layered.spacing.nodeNodeBetweenLayers": orientation === "DOWN" ? "90" : "135",
    "elk.spacing.nodeNode": orientation === "DOWN" ? "20" : "50",
```